files/image_config/ntp/ntp-systemd-wrapper

```diff
@@ -27,21 +28,29 @@ fi
 
 (
         flock -w 180 9
+        ntpEnabled=$(/usr/local/bin/sonic-cfggen -d -v 'NTP["global"]["admin_state"]' 2> /dev/null)
+        if [ "$ntpEnabled" = "disabled" ]
```

Comment:
Can `ExecCondition=` in the systemd service file be used to check this condition instead? That way the service status will accurately reflect that a condition check failed.

There also shoudn't be another ntp daemon running at this point of time, so there shouldn't be anything that needs to be stopped.

Reply:
hmm. nice idea, I guess it will work. I will try it and let you know the result here

Reply:
Any update here on whether this worked?

Reply:
Hey. I tried it with another daemon and it actually works fine. I tried to rework it here, but it won't work here in a couple of reasons:
1. It is not a one-line solution and will basically make this implementation more complex.
2. We are using lock here, which will be much harder to use in the service file.
3. We need to use `ExecStopPost` condition to stop the daemon.

Other reasons I don't like this option:
1. We need to modify ntp service file which is controlled by open source community and not modified by us.
2. That `ntp-systemd-wrapper` script was created exactly to customize the control of ntp daemon.

So I would like to avoid of such changes